### PR TITLE
[WV][000000] Allow trailing comma

### PR DESCRIPTION
**Pull request by warrenv**
Require dangling comma on multiline.

### CODE REVIEW

**Comment by aribouius** [comment]
Is the intention of this PR to _allow_ trailing commas, or _require_ them?

This config inherits from the [airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/legacy.js#L4) config, which defines this rule, and as such we'll need to retain the rule in our config to implement a change.

**Comment by warrenv** [comment]
Prior to this PR. This fails linting:

```javascript
{
  a: 1,
  b: 2,
}
```

When this PR is applied, linting passes.

**Comment by aribouius** [comment]
Hmm that's a bit odd, given we directly copy pasted `airbnb`'s config, and theirs disallows trailing commas.  We had a JS style working group that actually voted on this rule, and it was decided to disallow trailing commas (hence the rule definition in our config).  Thus it would appear our style guide was not updated to reflect properly.

**Comment by warrenv** [comment]
I don't recall the discussion, but the benefits are well discussed and convincing. How do we proceed? Where can the discussion be reopened (I tried in the slack engineering room)?

**Comment by aribouius** [comment]
I must have missed it in the engineering room.  Our rules are always open for modification, but should reflect the opinion of the majority.  How about taking another poll inside the `uiux_eng` room?

**Comment by colinrymer** [comment]
I'm in favor of both allowing and requiring trailing commas. Anything that helps reduce unintentional bugs and keeps diffs as relevant as possible is a win in my book.

**Comment by warrenv** [comment]
@colinrymer so requiring dangling comma in multiline only would work for you?

**Comment by colinrymer** [comment]
I think that a trailing comma in a single line statement is a bit annoying, but in a multiline, I'm all for it.

File: rules/style.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 module.exports = {
   rules: {
-    // disallow trailing commas in object literals
-    'comma-dangle': [2, 'never'],
+    'comma-dangle': [2, 'always-multiline'],
```
Comment (aribouius) [inline rules/style.js]: Can we keep the comment, but just update it?

**Comment by c0** [comment]
👍

**Comment by aribouius** [comment]
👍

**Pull request merged**